resource-timing/no-entries-for-cross-origin-css-fetched.sub.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Make sure that resources fetched by cross origin CSS are not in the timeline.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<link rel=stylesheet id=cross_origin_style href="//{{domains[www1]}}:{{ports[http][1]}}{{location[path]}}/../resources/nested.css">
```

Comment:
can you please add a comment as to what the `href` resolves to here?

Reply:
added